app/models/rich_text_reaction.rb

```diff
@@ -0,0 +1,53 @@
+# == Schema Information
+#
+# Table name: rich_text_reactions
+#
+#  id           :bigint           not null, primary key
+#  emoji        :string           not null
+#  created_at   :datetime         not null
+#  updated_at   :datetime         not null
+#  rich_text_id :bigint           not null
+#  user_id      :bigint           not null
+#
+# Indexes
+#
+#  index_rich_text_reactions_on_rich_text_id  (rich_text_id)
+#  index_rich_text_reactions_on_user_id       (user_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (rich_text_id => action_text_rich_texts.id)
+#  fk_rails_...  (user_id => users.id)
+#
+class RichTextReaction < ApplicationRecord
+  # Permissible set of emojis for standup updates and comments.
+  EMOJIS = {
+    👍: 'thumbs-up',
+    🤔: 'thinking',
+    🎉: 'hooray',
+    🤷: 'shrug',
+    👎: 'thumbs-down',
+    👀: 'eyes'
+  }.freeze
```

Comment:
The emoji and the caption will be flipped. After testing the code in the console, I’ve come to the realization is that a the caption being the key makes it easier for development.
i.e. It’s easier and more readable to type `EMOJIS[:thumbs_up]` then it is to type `EMOJIS[:👍]`.